samples/react-native-app/App.js

```diff
@@ -9,7 +10,9 @@ const instructions = Platform.select({
 export default function App() {
   return (
     <View style={styles.container}>
-      <Text style={styles.welcome}>Welcome to React Native!</Text>
+      <Container>
```

Comment:
technically these samples have been broken for web for a while now. as of react native web 12.0.0 components like "View" and "Text" no longer forward arbitrary props. I created a <Container> element to get around this issue.

https://github.com/necolas/react-native-web/issues/1146

Reply:
![Uploading image.png…]()